gammapy/maps/wcs.py

```diff
@@ -882,6 +882,20 @@ def __repr__(self):
         )
         return str_
 
+    def __eq__(self, other):
+        """Test equality between two `~gammapy.maps.WcsGeom`. Espect tolerance of 1e-6 for `~astropy.wcs.WCS` objects"""
+        if not isinstance(other, self.__class__):
+            raise TypeError('Cannot compare WcsGeom with {}'.format(other.__class__))
```

Comment:
I think you're supposed to `return NotImplemented` when implementing `__eq__`, no?
Can you please check the errors you get and check the Python docs what's recommended?

Reply:
OK. Done